js/pro/coinex.js

```diff
@@ -455,18 +457,17 @@ module.exports = class coinex extends coinexRest {
             throw new NotSupported (this.id + ' watchOrderBook() aggregation must be one of ' + aggregations.join (', '));
         }
         params = this.omit (params, 'aggregation');
+        const watchOrderBookSubscriptions = this.safeValue (this.options, 'watchOrderBookSubscriptions', {});
+        watchOrderBookSubscriptions[symbol] = [ market['id'], limit, aggregation, true ];
         const subscribe = {
-            'method': 'depth.subscribe',
+            'method': 'depth.subscribe_multi',
             'id': this.requestId (),
-            'params': [
-                market['id'],
-                limit,
-                aggregation,
-                true,
-            ],
+            'params': Object.values (watchOrderBookSubscriptions),
         };
+        this.options['watchOrderBookSubscriptions'] = watchOrderBookSubscriptions;
+        const subscriptionHash = this.hashMessage (this.json (watchOrderBookSubscriptions));
```

Comment:
@pcriadoperez hum `hashMessage`does not exist in python

Reply:
`this.hash (this.encode (this.json (watchOrderBookSubscriptions)));` should do it tho, going to fix it